src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_scenario.py

```diff
@@ -203,14 +203,14 @@ def test_container_acr_env_var(self, resource_group):
         ])
 
         # Update Container App with ACR
-        update_string = 'containerapp update -g {} -n {} --min-replicas 0 --max-replicas 1 --set-env-vars testenv=testing'.format(
+        update_string = 'containerapp update -g {} -n {} --min-replicas 1 --max-replicas 1 --set-env-vars testenv=testing'.format(
             resource_group, containerapp_name)
         self.cmd(update_string, checks=[
             JMESPathCheck('name', containerapp_name),
             JMESPathCheck('properties.configuration.registries[0].server', registry_server),
             JMESPathCheck('properties.configuration.registries[0].username', registry_username),
             JMESPathCheck('length(properties.configuration.secrets)', 1),
-            JMESPathCheck('properties.template.scale.minReplicas', '0'),
+            JMESPathCheck('properties.template.scale.minReplicas', '1'),
```

Comment:
In behind will update the --min-replicas to 0, need to set the --min-replicas to other value first.